src/journal/journalctl.c

```diff
@@ -1572,6 +1582,22 @@ static int add_syslog_identifier(sd_journal *j) {
         return 0;
 }
 
+static int add_exclude_identifier(sd_journal *j) {
+        int r;
+
+        assert(j);
+
+        j->exclude = set_free_free(j->exclude);
+
+        STRV_FOREACH(i, arg_exclude_identifier) {
+                r = set_put_strdup_full(&j->exclude, &string_hash_ops, *i);
```

Comment:
Why `string_has_ops` is used instead of `string_has_ops_free`??

Please use `set_put_strdupv()`, and use `set_free()` in the above.

Reply:
Done